core/src/main/java/org/frankframework/pipes/JsonPipe.java

```diff
@@ -161,14 +172,23 @@ public void setVersion(String version) {
 
 	/**
 	 * When direction is JSON2XML, it wraps a root element around the converted message.
-	 * When direction is XML2JSON, it includes the name of the root element as a key in the converted message.
+	 * When direction is XML2JSON, it includes the name of the root element as a key in the converted message, thus preserving the structure of the original input message.
 	 * @ff.default TRUE when JSON2XML and FALSE when XML2JSON
 	 */
 	public void setAddXmlRootElement(boolean addXmlRootElement) {
 		this.addXmlRootElement = addXmlRootElement;
 	}
 
-	/** Format the output in easy legible way (currently only for XML) */
+	/**
+	 * When direction is JSON2XML, specifies the name of the root element when {@literal addXmlRootElement} is {@code true}.
+	 * When direction is XML2JSON, can not be used.
+	 */
+	@Default(DEFAULT_ROOT_ELEMENT_NAME)
```

Comment:
I don't know which is preferred officially: '@Default` annotation, or `@ff.default` in the JavaDoc.
Personally I'd prefer the use of `@ff.default` in the JavaDoc, which is also more commonly used.

Reply:
The annotation is actually way more useful IMO. It allows me to reference the value of the constant, which is not possible with `@ff.default`. This prevents the documentation from becoming out of sync with the actual values used within the code.